tasks/common/protocol-versions.md

```diff
@@ -0,0 +1,62 @@
+# `ProtocolVersions` State Changes
+
+The `ProtocolVersions` contract signals the _recommended_ and _required_ protocol versions to nodes.
+Halting on incompatible protocol vesions is currently opt-in.
+
+It has two functions `setRecommended` and `setRequired` to set both versions individually.
+We usually aim to set the _recommended_ version *2 weeks* before a protocol upgrade and the _required_
+version *2 days* before.
+
+## Deployments
+
+* Mainnet: [`0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`](https://etherscan.io/address/0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A)
+* Sepolia: [`0x79ADD5713B383DAa0a138d3C4780C7A1804a8090`](https://sepolia.etherscan.io/address/0x79ADD5713B383DAa0a138d3C4780C7A1804a8090)
```

Comment:
We should instead link to these in the superchain registry, since that's the source of truth. The below makes this change + fixes the mainnet address
```suggestion
* Mainnet: [`0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935`](https://github.com/ethereum-optimism/superchain-registry/blob/67128af695e57d6e32a740f8587aca4e0bded888/superchain/configs/mainnet/superchain.yaml#L7)
* Sepolia: [`0x79ADD5713B383DAa0a138d3C4780C7A1804a8090`](https://github.com/ethereum-optimism/superchain-registry/blob/67128af695e57d6e32a740f8587aca4e0bded888/superchain/configs/sepolia/superchain.yaml#L7)
```